torch_xla/__init__.py

```diff
@@ -116,11 +116,27 @@ def _summarize_fn_tracker():
 
 def _aws_ec2_inf_trn_init():
   try:
-    from torch_neuronx import xla
+    from libneuronxla.libneuronpjrt_path import libneuronpjrt_path
   except ImportError:
-    return
+    pass
   else:
-    xla.init()
+    os.environ.setdefault('PJRT_DEVICE', 'NEURON')
+    os.environ.setdefault('NEURON_LIBRARY_PATH', libneuronpjrt_path())
```

Comment:
You can put the default path in the `NeuronPlugin` here: https://github.com/pytorch/xla/blob/f45775a5aeafb5219a045ab96d240b89c4106f42/torch_xla/_internal/neuron.py#L32

Long-term, you should put that plugin in your `libneuronxla` package and [register it as an `entry-point`](https://github.com/pytorch/xla/blob/master/docs/plugins.md#pytorchxla-plugin-package) so  you don't have to upstream Neuron-specific config changes. We can follow up on that after the release.

Reply:
I put the default path in [xla/torch_xla/_internal/neuron.py](https://github.com/pytorch/xla/blob/f45775a5aeafb5219a045ab96d240b89c4106f42/torch_xla/_internal/neuron.py#L32) also now, but I need NEURON_LIBRARY_PATH setting here for proper Neuron Cache operations.